### DISCUSSION TITLE
PHP/JS README.md

### DISCUSSION THREAD

**Discussion by SheiylaDev** [community/community#61522] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I would like to know if we can add php or JavaScripts to the profile page ?

**Comment by GhostOf0days** [comment]
No, but you can create an image using CSS and HTML then serve it to a Vercel deployment. Use the Vercel deployment link in your ReadMe to serve the content, and you can make your ReadMe dynamic without having to add new commits to it.

**Comment by SheiylaDev** [comment]
Thank you for the answer I suspected a little but I want confirmation !
